subsys/partition_manager/CMakeLists.txt

```diff
@@ -4,6 +4,13 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+if (NOT CONFIG_FLASH_MAP_CUSTOM)
```

Comment:
We actually don't know if `CONFIG_PARTITION_MANAGER_ENABLED` is set at this point. So you should add that to this check.

`if (CONFIG_PARTITION_MANAGER_ENABLED AND NOT CONFIG_FLASH_MAP_CUSTOM)`